packages/jest-test-sequencer/src/index.ts

```diff
@@ -46,6 +52,14 @@ type ShardPositionOptions = ShardOptions & {
 export default class TestSequencer {
   private readonly _cache = new Map<TestContext, Cache>();
 
+  protected readonly globalConfig: Config.GlobalConfig;
+  protected readonly contexts: Array<TestContext>;
+
+  constructor({contexts, globalConfig}: TestSequencerOptions) {
+    this.globalConfig = globalConfig;
+    this.contexts = contexts;
+  }
```

Comment:
Don't have a ton of experience with typescript, so want to make sure this is the correct appraoch